app/models/organisation.rb

```diff
@@ -489,7 +489,7 @@ def has_published_publications_of_type?(publication_type)
   end
 
   def has_services_and_information_link?
-    organisations_with_services_and_information_link.include?(slug)
+    return false
```

Comment:
Has rubocop not copped you for that return??

Reply:
Ah it has! 😂

Reply:
Old habits die hard ;)